RadixWallet/Clients/OverlayWindowClient/OverlayWindowClient+Interface.swift

```diff
@@ -78,29 +85,17 @@ extension OverlayWindowClient {
 			case emailSupport(additionalInfo: String)
 		}
 
+		public struct SheetState: Sendable, Hashable, Identifiable {
+			public let id = UUID()
+			public let image: ImageAsset?
+			public let text: String
```

Comment:
Very specific content for a generic sheet, I would opt implementing something to the FullScreenOverlayCoordinator.State.